zproject/prod_settings_template.py

```diff
@@ -227,6 +227,13 @@
 ## (defined in AUTH_LDAP_USER_SEARCH).
 # AUTH_LDAP_USERNAME_ATTR = "uid"
 
+## TODO: <explain the feature here>
+# LDAP_EXTERNAL_AUTH_ID_ENABLED = True
```

Comment:
I think we can probably drop this setting, and just compute it from whether `LDAP_ATTR_NAME_FOR_EXTERNAL_AUTH_ID` is null if application code would like to have a boolean value. (Checking `LDAP_ATTR_NAME_FOR_EXTERNAL_AUTH_ID is None` seems fine though).

I think `AUTH_LDAP_EXTERNAL_AUTH_ID_ATTR` might be a more consistent name though?

Oh, or maybe could this just be an optional field in `AUTH_LDAP_USER_ATTR_MAP`, maybe called `unique_id`?

Reply:
okay yeah, let's fold this into `AUTH_LDAP_USER_ATTR_MAP`. done now.